index.rst

```diff
@@ -42,20 +42,18 @@
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
+Introduction
+============
 
-.. note::
+Many of the control system components, but not all, have been developed for deployment via Docker containers. We currently leverage a `docker-compose` mechanism for container deployment to host services. With the proliferation of Kubernetes (k8s) clusters around the Vera C. Rubin Observatory and the associated sites, a new style of deployment and configuration is needed. We will leverage the experience SQuaRE has gained deploying the EFD and Nublado services and use a similar system based on Helm charts and ArgoCD configuration and deployment. The following sections will discuss the two key components and then will be expanded into including operation aspects as experience on that front is gained.
```

Comment:
Long time ago Tim Jenness suggested to break the lines around 80 characters so, in the future, if you need to change part of a paragraph it would not look like you changed the entire paragraph. I think it would be good if you could do something similar with this document. 

Also, on this paragraph, you mention,

> We currently leverage a `docker-compose` mechanism for container deployment to host services. 

I think this is about to become history pretty soon. I would recommend changing it to something like:

> Initially we leveraged `docker-compose` as a mechanism for prototyping container deployment to host services.